jobs/aws/eks-anywhere-build-tooling/autoscaler-1-21-presubmits.yaml

```diff
@@ -44,18 +50,15 @@ presubmits:
           if $(make check-project-path-exists); then make build -C $PROJECT_PATH; fi
         env:
         - name: PROJECT_PATH
-          value: projects/kubernetes/autoscaler
-        - name: PRUNE_BUILDCTL
-          value: "true"
+          value: "projects/kubernetes/autoscaler"
         - name: RELEASE_BRANCH
           value: "1-21"
+        - name: PRUNE_BUILDCTL
+          value: "true"
         resources:
           requests:
             memory: "16Gi"
             cpu: "4"
-          limits:
```

Comment:
I requested that remove all limits from presubmits since they are not needed due to the nature of fargate. Fargate supplies a node of the exact size you request in your resources.request and limits are not applied.